server/viame_server/viame.py

```diff
@@ -152,3 +156,8 @@ def update_attribute(self, data, attribute, params):
     @autoDescribeRoute(Description("").modelParam("id", model=Attribute, required=True))
     def delete_attribute(self, attribute, params):
         return Attribute().remove(attribute)
+
+    @access.user
+    @describeRoute(Description("Get Valid Image/Video filetypes for uploading"))
+    def get_valid_filetypes(self, params):
+        return {'image': validImageFormats, 'video': validVideoFormats, 'web': webValidImageFormats}
```

Comment:
We use double quotes throughout the codebase, and more generally, code is formatted with [black](https://black.readthedocs.io/en/stable/the_black_code_style.html). Formatting this with black gives me the following (also added an EOF newline):

```suggestion
        return {
            "image": validImageFormats,
            "video": validVideoFormats,
            "web": webValidImageFormats,
        }

```